### PR TITLE
Use F_NOEXPORTRUNTIME if __rtti_base is missing

### LINKED ISSUE #1807: Loader __getArray, "unsupported align: -1"
I'm following along the [Loader documentation examples](https://www.assemblyscript.org/loader.html#reading-arrays) about reading arrays from AssemblyScript. While trying to implement the example exactly as in the documentation, calling `__getArray` produces the error "unsupported align: -1". 

An minimal project reproducing the problem is here:
https://github.com/data-ux/unsupported-align

I have tried importing the loader both from node_modules and from jsdelivr with the same effects. I'm on Assemblyscript version 0.18.25

### PR DESCRIPTION
**Pull request by MaxGraey**
Fix #1807

- [x] I've read the contributing guidelines

### CODE REVIEW

File: lib/loader/index.js [hunk 1]
```diff
@@ -88,7 +88,7 @@ function postInstantiate(extendedExports, instance) {
   const __pin = exports.__pin || F_NOEXPORTRUNTIME;
   const __unpin = exports.__unpin || F_NOEXPORTRUNTIME;
   const __collect = exports.__collect || F_NOEXPORTRUNTIME;
-  const __rtti_base = exports.__rtti_base || ~0; // oob if not present
+  const __rtti_base = exports.__rtti_base || F_NOEXPORTRUNTIME;
```
Comment (dcodeIO) [inline lib/loader/index.js:91]: `__rtti_base` is a pointer into memory (unlike those above, which are functions), while `F_NOEXPORTRUNTIME` is a function. What would be the error here?

Comment (MaxGraey) [reply lib/loader/index.js:91]: yeah, just figure out this. Need other approach

File: lib/loader/index.js [hunk 2]
```diff
@@ -98,9 +99,10 @@ function postInstantiate(extendedExports, instance) {
   /** Gets the runtime type info for the given id. */
   function getInfo(id) {
     const U32 = new Uint32Array(memory.buffer);
-    const count = U32[__rtti_base >>> 2];
+    const rtBase = rttiBase();
+    const count = U32[rtBase >>> 2];
```
Comment (dcodeIO) [inline lib/loader/index.js:103]: Could use `rttiBase()` on this line to perform a check once, and `__rtti_base` below

**Review by dcodeIO — APPROVED** [review]
LGTM! Feel free to merge :)

**Pull request merged**